homeassistant/components/vesync/strings.json

```diff
@@ -49,6 +49,17 @@
         }
       }
     },
+    "humidifier": {
+      "vesync": {
+        "state_attributes": {
+          "mode": {
+            "state": {
+              "humidity": "Humidity"
```

Comment:
But what does mode humidity mean? It's a humidifier, so I would expect any mode that isn't off to do something with Humidity, but what?

Reply:
Humidity and auto. 

![image](https://github.com/user-attachments/assets/3bb26135-a174-40a8-9cee-754d5aac27ea)

Reply:
Yup!  My goal is to align modes and other details to the native app where it doesn't align nicely to HA's OOTB modes etc.   After this PR other items are needed for the 6000s to expand function but figured this moves in the right direction and gets tests in.